README.md

```diff
@@ -125,7 +108,22 @@ superwerker features can be activated individually. This enables a gradual roll-
 
 ### How do I receive updates?
 
-We plan to roll-out releases via GitHub releases. The update is then deployed via the current CloudFormation template.
```

Comment:
The update instructions are now listed in the FAQ, while the installation instructions live in their own section. I think we should make this more consistent., e.g. introduce an "updates" section as we have for the [installation section](#installation) and link to this from the FAQ. What do you think?